workbench/src/main/setupAddPlugin.js

```diff
@@ -41,17 +41,17 @@ export default function setupAddPlugin() {
         const envName = `invest_plugin_${pluginID}`;
         const mamba = settingsStore.get('mamba');
         execSync(`${mamba} create --yes --name ${envName} -c conda-forge "python<3.12" "gdal<3.6"`,
-          { stdio: 'inherit' });
+          { stdio: 'inherit', windowsHide: true });
         logger.info('created mamba env for plugin');
         execSync(`${mamba} run --name ${envName} pip install "git+${pluginURL}"`,
-          { stdio: 'inherit' });
+          { stdio: 'inherit', windowsHide: true });
         logger.info('installed plugin into its env');
 
         // Write plugin metadata to the workbench's config.json
-        const envInfo = execSync(`${mamba} info --envs`).toString();
+        const envInfo = execSync(`${mamba} info --envs`, { windowsHide: true }).toString();
         logger.info(`env info:\n${envInfo}`);
-        const envPath = envInfo.match(`${envName}\\s+(.+)\n`)[1];
-        logger.info(`env path: ${envPath}`);
+        const envPath = envInfo.match(`${envName}\\s+(.+)$`)[1];
```

Comment:
I think the end matching character is `$`. I will admit I didn't test this small change yet.